dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,103 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/#python"
+COPYRIGHT="2003-2006 Red Hat Inc.
+           2003 David Zeuthen
+           2004 Rob Taylor
+           2004 Anders Carlsson
+           2005 Colin Walters
+           2005-2019 Collabora Ltd.
+           2016 Simon McVittie"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$portVersion.tar.gz"
+CHECKSUM_SHA256="b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
+SOURCE_DIR="dbus-python-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dbuspython = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libdbus_1
+	lib:libglib_2.0
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	lib:libglib_2.0
+	lib:libdbus_1
+	devel:libglib_2.0
+	devel:libdbus_1
+	"
+
+PYTHON_PACKAGES=(python python36 python3)
+PYTHON_VERSIONS=(2.7 3.6 3.7)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersion=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku\n\
+	cmd:python$pythonVersion\
+	\""
+BUILD_REQUIRES="$BUILD_REQUIRES
+	setuptools_$pythonPackage"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:python$pythonVersion\n\
+	cmd:autoconf\n\
+	cmd:gawk\n\
+	cmd:gcc\n\
+	cmd:bash\n\
+	cmd:pkg_config\n\
+	cmd:aclocal\n\
+	cmd:libtoolize\n\
```

Comment:
For these, please create a `BUILD_PREREQUIRES` outside of the loop, as they are not affected by the python version change.